src/app/routes/enrolment/models/enrolment-claim.ts

```diff
@@ -15,6 +16,8 @@ export class EnrolmentClaim
   organization: string;
   application?: string;
   requestDate: Date;
+  expirationStatus: string;
```

Comment:
```suggestion
  expirationStatus?: ExpirationStatus;
```
as this might not null/undefined.